lib/js_routes.rb

```diff
@@ -19,7 +19,8 @@ class JsRoutes
     url_links: nil,
     camel_case: false,
     default_url_options: {},
-    compact: false
+    compact: false,
+    serializer: "this.default_serializer"
```

Comment:
@bogdan If we set the default to `nil` here, we will have to do a check somewhere else to set the default initializer. It has to get set somewhere, we can't just have a `nil` initializer floating around. Where would you prefer it get set?

Something like this on line 108?

    js.gsub!("SERIALIZER", @options[:serializer] || "this.default_serializer")

I think setting in the defaults is better than this option. This feels like a magic value. And really, `this.default_serializer` _is_ the default—not `nil`. It doesn't make sense to have a `nil` initializer.